inc/deploypackage.class.php

```diff
@@ -146,7 +146,7 @@ public function getSpecificMassiveActions($checkitem = null)
     {
 
         $actions = [];
-        if (strstr($_SERVER["HTTP_REFERER"], 'deploypackage.import.php')) {
+        if (isset($_SERVER["HTTP_REFERER"]) && strstr($_SERVER["HTTP_REFERER"], 'deploypackage.import.php')) {
```

Comment:
Maybe less difficult to read, and should achieve the same result:
```suggestion
        if (strstr($_SERVER["HTTP_REFERER"] ?? '', 'deploypackage.import.php')) {
```